store/sqlstore/file_info_store.go

```diff
@@ -44,6 +44,16 @@ type fileInfoWithChannelID struct {
 	Archived        bool
 }
 
+// Find the end of the term and include the wildcard if it exists
+// because we will put a wildcard at the end of the term regardless of whether a wildcard exists or not
+var wildcardRegExpForFileSearch = regexp.MustCompile(`\*?$`)
+
+// Find all characters between quotes
+var exactPhraseRegExpForFileSearch = regexp.MustCompile(`"[^"]+"`)
+
```

Comment:
Similar comment as in the other file:
```suggestion
// Regexp to match an asterisk at the end of the string. It is used
// in the file search logic to put a wildcard at the end of the search
// term regardless of whether a wildcard exists or not
var wildcardRegExpForFileSearch = regexp.MustCompile(`\*?$`)

// Regexp to mathch any quoted string. It is used in the file search
// logic to avoid adding a wildcard at the end of the quoted search
// terms, which need to be matched literally.
var exactPhraseRegExpForFileSearch = regexp.MustCompile(`"[^"]+"`)

```